lib/client.js

```diff
@@ -816,7 +798,11 @@ function onSocketError (err) {
 }
 
 function onSocketData (data) {
-  this[kParser].execute(data)
+  try {
```

Comment:
I see a small drop in the benchmarks, I'll run them again to be sure the use of the `try`/`catch` is having an impact.

https://github.com/nodejs/undici/runs/2275815074?check_suite_focus=true

Reply:
Removing the try catch should be easy. I’ll fix it.

Reply:
On a second run, benchmarks look good. I would not change it yet.